packages/smart-contracts/packages/nft/scripts/data/networkConfig.json

```diff
@@ -39,7 +39,7 @@
         "nft_symbol": "WOWPASS",
         "vesting_contract": "",
         "vesting_pool_id": 0,
-        "max_level": 5,
+        "max_level": 10,
```

Comment:
Why is this changing? Never heard that we will have 10 nft levels

Reply:
but we are adding genesis levels, thus it goes up (genesis level id 6 = level 1, genesis id 7 = level 2)

Reply:
But it doesn't work like that, we only have 5 level all the time